.github/workflows/ci-build-docs.yml

```diff
@@ -0,0 +1,17 @@
+name: Build docs
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'docs/sources/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build docs
+        run: yarn docs:lint
```

Comment:
@knylander-grafana @jdbaldry which command should we execute to verify that the docs can be built?

Reply:
Here's how I do it in Writers' Toolkit: https://github.com/grafana/writers-toolkit/blob/main/.github/workflows/validate-documentation.yml#L59-L65

Reply:
> Here's how I do it in Writers' Toolkit: https://github.com/grafana/writers-toolkit/blob/main/.github/workflows/validate-documentation.yml#L59-L65

I've added it, is the build output the one expected: https://github.com/grafana/explore-profiles/actions/runs/9990850582/job/27612399685?pr=59 ?

There are a lot of warnings: 
<img width="1071" alt="image" src="https://github.com/user-attachments/assets/f5c4a63d-d6a1-45a2-9276-b2db199bc46d">

Reply:
Yeah, unfortunately that's expected as we only build a subset of the site and some references are not found in that subset. 

The main thing is that if this command exits 0, then there were no build errors that would stop a production build in the website repository. We don't want to sync any docs that would stop a production build because it breaks the whole site build, so no new changes can be published until that's fixed

Reply:
Cool, thx for the clarification